packages/graph-explorer/src/modules/SearchSidebar/QuerySearchTabContent.tsx

```diff
@@ -160,6 +160,9 @@ function QueryTextArea({
       className={cn("h-full min-h-[5lh] w-full font-mono text-sm", className)}
       placeholder={queryPlaceholder}
       onKeyDown={onKeyDown}
+      spellCheck={false}
```

Comment:
```suggestion
      spellCheck="false"
```
nit: seems like a string value would also work

Reply:
If we use form input, we should consider disabling the spell-check and auto-complete for the entire form rather than specific text fields.  Although, maybe I'm wrong with that and maybe someone would want a spellchecker for something specific...